plugins/polio/js/src/forms/RoundVaccinesForm.tsx

```diff
@@ -150,6 +152,34 @@ export const RoundVaccinesForm: FunctionComponent<Props> = ({
         vaccines.length,
     ]);
 
+    // Fill in number of doses with default value when adding vaccine
+    useEffect(() => {
+        vaccines.forEach((vaccine, index) => {
+            const dosesPerVial =
+                rounds[roundIndex].vaccines[index].doses_per_vial;
+            // I'm sorry for this, I really had to null check this value
```

Comment:
😄

Reply:
This is the most meme-able line of js I ever wrote